Codec/Archive/Zip.hs

```diff
@@ -350,16 +358,16 @@ checkEntry s = do
 -- | Unpack entire archive into specified directory. The directory will be
 -- created if it does not exist.
 
-unpackInto :: Path b Dir -> ZipArchive ()
-unpackInto dir' = do
+unpackInto :: Bool -> Path b Dir -> ZipArchive ()
```

Comment:
How that we have more arguments, we should add `-- ^` per-argument descriptions like we have for `saveEntry`.